emissionsapi/openapi.yml

```
@@ -70,6 +70,35 @@ paths:
                 $ref: '#/components/schemas/Averages'
         400:
           description: Parameter error
+  /api/v1/statistics.json:
+    get:
+      operationId: emissionsapi.web.get_statistics
+      description: |
+          Get statistical values for specified time intervals.
+
+          The parameters `geoframe`, `country` and `polygon` overwrite each other so that only one will be used even if multiple parameters are provided.
+          The order of parameter precedence is:
+            1. geoframe
+            2. country
+            3. polygon
+      parameters:
+        - $ref: '#/components/parameters/interval'
+        - $ref: '#/components/parameters/geoframe'
+        - $ref: '#/components/parameters/country'
+        - $ref: '#/components/parameters/polygon'
+        - $ref: '#/components/parameters/begin'
+        - $ref: '#/components/parameters/end'
+        - $ref: '#/components/parameters/limit'
+        - $ref: '#/components/parameters/offset'
+      responses:
+        200:
+          description: Array of statistical data about carbon monoxide for specified intervals.
```

Isn't interval here singular? Since you only specify one interval for a call. Maybe something like this would fit better?

```suggestion
          description: Array of statistical data about carbon monoxide grouped by specified interval.
```

Or maybe you have another idea how to describe it.

---

No, interval**s** is correct. Users specify multiple intervals by providing an interval length (e.g. `day`)